coderd/rbac/role.go

```diff
@@ -17,7 +17,9 @@ type Permission struct {
 // Users of this package should instead **only** use the role names, and
 // this package will expand the role names into their json payloads.
 type Role struct {
-	Name        string       `json:"name"`
+	Name string `json:"name"`
+	// DisplayName is used for UI purposes. If the role has no display name,
+	// that means the UI should never display it.
 	DisplayName string       `json:"display_name"`
```

Comment:
Can we just not send this role to the UI at all? I think the whole `org-member:` thing is just the Rego leaking out a bit.

Reply:
Ah gotcha.